internal/app/wwctl/server/status/main.go

```diff
@@ -2,10 +2,9 @@ package status
 
 import (
 	"github.com/hpcng/warewulf/internal/pkg/warewulfd"
-	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
 func CobraRunE(cmd *cobra.Command, args []string) error {
-	return errors.Wrap(warewulfd.DaemonStatus(), "failed to start daemon")
+	return warewulfd.DaemonStatus()
```

Comment:
Why did you remove the error wrap? Unless you are using predefined errors (which should be avoided), it is good practice to wrap every error we return.